examples/client_realms/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "node-minecraft-protocol-example",
+  "version": "0.0.0",
+  "private": true,
+  "dependencies": {
```

Comment:
Remove
```suggestion
  {
     "name": "node-minecraft-protocol-example",
     "version": "0.0.0",
      "private": true,
      "description": "A node-minecraft-protocol example"
```

Reply:
This is the case in other examples so believe it should stay as is.

Reply:
ok